src/Api/SecretManagerFeatures/Models/Response/SecretIdentifierResponseModel.cs

```diff
@@ -29,5 +30,7 @@ public SecretIdentifierResponseModel(Secret secret, string obj = "secret")
     public DateTime CreationDate { get; set; }
 
     public DateTime RevisionDate { get; set; }
+
+    public Dictionary<System.Guid, string> ProjectGuids { get; set; }
```

Comment:
I think it's fine to do a separate API call to fetch the projects. Or we could include it in the actual response model instead of in each individual secret.

Reply:
I started making these changes to see what this would look like, It was getting messy and I stashed the changes - if you want to continue on this route we should go over my stash and see if it's what you imagined.